apps/asana/src/inngest/client.ts

```diff
@@ -13,6 +13,12 @@ export const inngest = new Inngest({
         offset?: string;
       };
     };
+    'token/refresh': {
+      data: {
+        organisationId: string;
+        refreshTokenInfo: string;
```

Comment:
```suggestion
        refreshToken: string;
```
You can replace `refreshTokenInfo` by `refreshToken` everywhere.

Reply:
import { refreshToken, type RefreshTokenResponseData } from '@/connectors/auth';

As you can see, I already created the same named function. It conflicts.

Reply:
Ok, no problem.